### DISCUSSION TITLE
Icon::make() error

### DISCUSSION THREAD

**Discussion by devemio** [filamentphp/filament#233] [Help] [answered]
I try to use the `Icon` field as shown in the documentation, but I get the following error:

```
Call to undefined function heroicon-s-check-circle()
```

Usage:
```php
public static function table(Table $table): Table
{
    return $table
        ->columns([
            Columns\Icon::make('status')
                ->options([
                    'accepted' => 'heroicon-s-check-circle',
                    'declined' => 'heroicon-x-circle',
                    'pending' => 'heroicon-s-clock',
                ]),
        ]);
}
```

Can you please advise?

**Comment by ryangjchandler** [comment, marked as answer]
The documentation is actually wrong, I'll try and sort this out tomorrow. 

The keys in the array should be the name of the icons and the values should be callback functions that return a boolean. 

If the callback returns true, that icon (key) will be used. 

The callback function receives the value of the column, pulled from the record.

**Reply by devemio** [reply]
Got it, thanks.
